index.html

```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
+                  <button>bootstrap</button>
+                  <button>Ruby</button>
+                </div>
+                <div class="see-project">
+                  <button class="large-button">see project</button>
+                </div>
+              </div>
+            </div>
+
+            <div class="work-2">
+              <div class="work-detail">
+                <div class="main-detail">
+                  <h4>professional art printing data</h4>
+                  <p class="paragraph">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                </div>
+                <div class="work-tools work-stack">
+                  <button>html</button>
+                  <button>bootstrap</button>
+                  <button>Ruby</button>
+                </div>
+                <div class="see-project">
+                  <button class="large-button">see project</button>
+                </div>
+              </div>
+            </div>
+
+            <div class="work-2">
+              <div class="work-detail">
+                <div class="main-detail">
+                  <h4>professional art printing data</h4>
+                  <p class="paragraph">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                </div>
+                <div class="work-tools work-stack">
+                  <button>html</button>
+                  <button>bootstrap</button>
+                  <button>Ruby</button>
+                </div>
+                <div class="see-project">
+                  <button class="large-button">see project</button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="skills">
+          <div class="all-skills">
+            <div class="skills-details">
+              <h1>About Me</h1>
+              <p>
+                Hello I'm a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+              </p>
+              <div class="details-btn">
+                <a
+                  href="./assets/images/CV - Blessing Oghenetega Ekiugbo (1).pdf"
+                  download
+                  >Get my Resume</a
+                >
+              </div>
+            </div>
+            <div class="tech-stack">
+              <div class="stack-heading">
+                <img
+                  src="./assets/icons/Rectangle 55.svg"
+                  alt="language-icon"
+                />
+                <h3>Language</h3>
```

Comment:
I suggest you change your skills section to a list of list items that is
the language, Framework and skills should also be in `<ul> <li>`

Reply:
@Nyame-Wolf Thank you for your review. I made the changes and now trying to re-submit for review, but cannot see the option for changes requested, just a direct submit. Can I send you the link here and you review it for me? Thank you

Reply:
@Nyame-Wolf Can you kindly review this before the day ends?

Reply:
@Nyame-Wolf https://github.com/Otegaa/setup-and-mobile-first/pull/2